components/ui/Button.tsx

```diff
@@ -13,7 +13,8 @@ export interface ButtonProps {
     | "pink"
     | "yellow"
     | "orange"
-    | "white";
+    | "white"
+    | "DialogComponent";
```

Comment:
We should be able to use the primary brand colour here, if that's not working we can discuss on a call

Reply:
Yeah.  I'd love to sync on that, because I want to see the button component work inside this component, and take care of the conflicting styles.  I'm def interested in troubleshooting that!